source/melpomene/README.md

```diff
@@ -0,0 +1,28 @@
+# Melpomene
+
+Melpomene is a desktop simulator for the MnemOS kernel.
+
+Rather than emulating any specific hardware, it runs the kernel in one thread, and userspace in a second thread.
+
+Drivers to implement platform-specific behavior, such as serial ports, are provided.
+
+## Running the simulator
```

Comment:
maybe worth mentioning that there's also a `cargo melpomene` alias, and that you can `cargo melpomene --help` to get a complete list of CLI options?

Reply:
Ah, I meant to do this! I even merged #21 so it made sense, then I totally forgot to do that. Will add.